indy_custom_comp_msgs/CompAlgorithm req
---
bool success